lib/matplotlib/artist.py

```diff
@@ -1203,7 +1203,8 @@ def _update_props(self, props, errfmt):
                     func = getattr(self, f"set_{k}", None)
                     if not callable(func):
                         raise AttributeError(
-                            errfmt.format(cls=type(self), prop_name=k))
+                            errfmt.format(cls=type(self), prop_name=k),
```

Comment:
The docstring could mention that formatting happens via named placeholders `{cls}` and `{prop_name}`.